src/model/trust.model.ts

```diff
@@ -78,6 +88,41 @@ interface TrustHistoricalBeneficialOwnerCommon {
   notified_date_year: string;
 }
 
+interface IndividualTrusteeCommon{
+  trusteeId?: string;
+  type: RoleWithinTrustType;
+  forename: string;
+  surname: string;
+  date_of_birth_day: string;
+  date_of_birth_month: string;
+  date_of_birth_year: string;
+  nationality: string;
+  second_nationality?: string;
+  property_name?: string;
+  address_line1: string;
+  address_line2: string;
+  city: string;
+  county: string;
+  country: string;
+  postal_code: string;
+  correspondence_address_line1?: string;
+  correspondence_property_name?: string;
+  correspondence_address_line2?: string;
+  correspondence_city?: string;
+  correspondence_county?: string;
+  correspondence_country?: string;
+  correspondence_postal_code?: string;
+}
```

Comment:
I don't think we need this as we need to map to the `TrustIndividual`  which  exists in this file. I think it needs to be a `type` rather than an `interface` though.